definitions/npm/express_v4.x.x/flow_v0.32.x-/test_response.js

```diff
@@ -1,33 +1,39 @@
 // @flow
 
-import express, { Router } from 'express';
-import http from 'http';
+import express, { Router } from "express";
+import http from "http";
 
 const app = express();
 
-app.use('/response_api', (req: express$Request, res: express$Response) => {
+app.use("/response_api", (req: express$Request, res: express$Response) => {
   const contentLength = String(2);
-  res.set('Content-Length', contentLength);
-  res.set('Content-Type', 'application/json');
+  res.set("Content-Length", contentLength);
+  res.set("Content-Type", "application/json");
   res.writeHead(200, {
-    'Content-Length': contentLength,
-    'Content-Type': 'application/json',
+    "Content-Length": contentLength,
+    "Content-Type": "application/json"
   });
-  res.end('{}');
+  res.end("{}");
   res.req;
 });
 
 // Can manually invoke router.handle() to handle a request.
 const router = new Router();
-app.use('/router', (req: express$Request, res: express$Response, next: express$NextFunction) => {
-  router.handle(req, res, next);
-});
-app.post('/post-router-callable', router);
+app.use(
+  "/router",
+  (req: express$Request, res: express$Response, next: express$NextFunction) => {
+    router.handle(req, res, next);
+  }
+);
+app.post("/post-router-callable", router);
 
 // Can use an express app directly as a server listener
 const httpServer = http.createServer(app);
 httpServer.listen(9000);
 
+// $ExpectError
+httpServer.listen("foo");
```

Comment:
`httpServer.listen("foo")` is valid due to `listen(path: string, callback?: (err?: ?Error) => mixed): Server;`. Try something like `httpServer.listen(true)` or `httpServer.listen([])`.